runtime/shared_common/shrinit.cpp

```diff
@@ -3952,6 +3953,26 @@ j9shr_getCacheSizeBytes(J9JavaVM *vm)
 	return ret;
 }
 
+/**
+ * Check if the shared cache is enabled by default or by using "-Xshareclasses" in the command line. 
+ * 
+ * @param [in] vm Pointer to the VM structure for the JVM
+ *
+ * @return TRUE if shared cache is enabled by default, FALSE otherwise
+ * 
+ */
+BOOLEAN
+j9shr_isSharedClassEnabledByDefault(J9JavaVM *vm)
```

Comment:
Given the description and the implementation below, this name seems wrong: `j9shr_isSharedClassEnabled` would be more appropriate. The VM field name should be updated to match.

Reply:
`j9 shr_isSharedClassEnabled` looks confusing to me. When shared class is enabled, there are 2 cases:
1. By default: only bootstrap classes are shared.
2. Using `-Xshareclasses` in the command line: Both bootstrap classes and non-bootstrap classes are shared.

I am fine with the current name or something like `j9shr_isOnlyBootClassesSharingEnable()`.

Reply:
I like `j9shr_isOnlyBootClassesSharingEnabled`, but perhaps an explanation of how this is expected to be used will help us agree on the right name.

Reply:
The query was created in response to this issue: "It would be useful to have a query to determine whether the SCC being used is an SCC created via the -Xshareclasses option, or the default SCC. One use case would be making heuristic decisions based on this information (eg #6734)." So I opted to have it return true if the SCC being used was the default and false in the -Xshareclasses case.

Reply:
It's possible in the future that boot classes won't be the only sharing enabled by default. Ideally platform class loader classes would also be shared by default.